src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_account_scenarios.py

```diff
@@ -373,3 +375,59 @@ def test_create_storage_account_with_files_aadds_true(self, resource_group):
 
         self.assertIn('azureFilesIdentityBasedAuthentication', result)
         self.assertEqual(result['azureFilesIdentityBasedAuthentication']['directoryServiceOptions'], 'AADDS')
+
+
+class RoleScenarioTest(LiveScenarioTest):
+    def run_under_service_principal(self):
+        account_info = self.cmd('account show').get_output_in_json()
+        return account_info['user']['type'] == 'servicePrincipal'
+
+
+@api_version_constraint(ResourceType.MGMT_STORAGE, min_api='2019-04-01')
+class RevokeStorageAccountTests(StorageScenarioMixin, RoleScenarioTest, LiveScenarioTest):
+    @ResourceGroupPreparer()
```

Comment:
I prefer adding `name_prefix='cli_test_storage_revoke` so that you could easily locate the rg.